client/src/features/dataConnectorsV2/components/DataConnectorsBoxListDisplay.tsx

```diff
@@ -33,23 +34,42 @@ import DataConnectorView from "./DataConnectorView";
 interface DataConnectorBoxListDisplayProps {
   dataConnector: DataConnector;
   dataConnectorLink?: DataConnectorToProjectLink;
+  extendedPreview?: boolean;
 }
 export default function DataConnectorBoxListDisplay({
   dataConnector,
   dataConnectorLink,
+  extendedPreview,
 }: DataConnectorBoxListDisplayProps) {
   const {
     name,
     description,
     visibility,
     creation_date: creationDate,
+    storage,
+    namespace,
   } = dataConnector;
 
   const [showDetails, setShowDetails] = useState(false);
   const toggleDetails = useCallback(() => {
     setShowDetails((open) => !open);
   }, []);
 
+  const type = `${storage?.configuration?.type?.toString() ?? ""} ${
+    storage?.configuration?.provider?.toString() ?? ""
+  }`;
+  const readOnly = storage?.readonly ? (
```

Comment:
This will prevent the element from being computed when `extendedPreview is false`.

```suggestion
  const readOnly = extendedPreview && (storage?.readonly ? (
```